extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -93,6 +93,8 @@ define('ext.wikia.adEngine.adContext', [
 	function updateAdContextRabbitExperiments(context) {
 		context.rabbits.ctpDesktop = isProperGeoAds('wgAdDriverCTPDesktopRabbitCountries');
 		context.rabbits.ctpMobile = isProperGeoAds('wgAdDriverCTPMobileRabbitCountries');
+		context.rabbits.queenDesktop = isProperGeoAds('wgAdDriverCTPDesktopQueenCountries');
+		context.rabbits.isQueenDesktopForced = window.location.href.indexOf('CTPQueenForce') > -1;
```

Comment:
How about this?
`context.rabbits.isQueenDesktopForced = isUrlParamSet('CTPQueenForce');`

Reply:
beautiful :)